client/src/Components/ShippingInformation.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+
+class ShippingInformation extends React.Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      shippingTime: [],
+      exclusions: [],
+      returnPolicy : [],
+      returnOverview : [],
+      giftWrapping: [],
+      getShippingCost: false,
+      shippingCost: [],
+      enteredShippingInfo: false,
+      zipCode : [],
+      shippingFrom: [],
+      shippingTo: [],
+      shippingToCountry: [],
+      deliveryDate : []
+    }
+    this.getShippingInformation = this.getShippingInformation.bind(this);
+    this.getReturnPolicy = this.getReturnPolicy.bind(this);
+    this.getShippingInfo = this.getShippingInfo.bind(this);
+    this.showShipping = this.showShipping.bind(this);
+    this.shippingCost = this.shippingCost.bind(this);
+    this.shippingToCountry = this.shippingToCountry.bind(this);    
+  }
+
+
+  componentDidMount(){
+    this.getShippingInformation()
+    this.getReturnPolicy()      
+  }
+  getShippingInformation(){
+    let manufacturingTime;
+    let shippingFrom = this.props.data[0]['shippingFrom']
```

Comment:
I noticed that `props.data[0]` is the only index of `data` that is every referenced; you should only pass a the single index of `data` that you need down as a prop, and in the process give it a more descript name beyond `data`